include/ogdf/basic/Graph_d.h

```diff
@@ -1100,6 +1100,34 @@ class OGDF_EXPORT Graph {
 		v->adjEntries.sort(newOrder);
 	}
```

Comment:
This method could just use the new iterator-based one (assuming that all our lists have `begin` and `end`), right? That would prevent some code duplication.